.github/workflows/changelog.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(github.event.pull_request.labels.*.name, 'skip-changelog')"
     env:
-      TOWNCRIER_VERSION: 22.12.0
+      TOWNCRIER_VERSION: 23.11.0
```

Comment:
This tricked my vision, because the minor version changed from 12 to 11, I thought it is a downgrade.